lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -809,6 +809,7 @@ def create_parameters(self):
         if not HAS_DOCKER_PY_3:
             create_params['cpu_shares'] = 'cpu_shares'
             create_params['volume_driver'] = 'volume_driver'
+            create_params['blkio_weight'] = 'blkio_weight'
```

Comment:
Why do you only add `blkio_weight` for `docker-py >= 3.0.0`? This should be supported since `docker-py == 1.9.0`, according to the [changelog](https://docker-py.readthedocs.io/en/stable/change-log.html).